cypress/e2e/4-tables/rowEditing.cy.js

```diff
@@ -0,0 +1,15 @@
+describe('Check functionality of Smart Table', ()=>{
+    context(' page - pages/tables/smart-table', ()=>{
+
+it ('Editing a row and checking if an item has been updated', ()=>{
+
+    cy.visit('pages/tables/smart-table');
+    cy.get('.nb-edit').eq(0).click();
+    cy.get('.form-control').eq(8).clear().type('test');
+    cy.get('.nb-checkmark').first().click();
+    cy.get('tbody tr').eq(0).find('td').eq(3).invoke('text', 'test').then((text) => {
+      });
```

Comment:
seems there is missed verification

Reply:
Yes, I need help because i don't understand about which missed verification you said) I will be so grateful if you could say this